src/main/java/org/ohdsi/authenticator/service/AccessTokenResolver.java

```diff
@@ -0,0 +1,45 @@
+package org.ohdsi.authenticator.service;
+
+import java.util.Optional;
+import java.util.function.Function;
+import org.apache.commons.lang3.StringUtils;
+
+public class AccessTokenResolver {
+
+    private static final String GOOGLE_IAP_JWT_HEADER = "x-goog-iap-jwt-assertion";
+    private static final String GOOGLE_IAP_AUTH_METHOD = "iap";
```

Comment:
Discussed on call: the hardcoded method name doesn't align to current method convention. What's more the method name doesn't seem to be utilized in IAP workflow at all. Need further thinking